source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+﻿/*
+This example shows how to perform Hand-Eye calibration.
+*/
+
+using System;
+using System.IO;
+using YamlDotNet.RepresentationModel;
+
+
+using MathNet.Numerics.LinearAlgebra;
+using MathNet.Numerics.LinearAlgebra.Double;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            printHeader("This example shows conversions to/from Transformation Matrix");
+
+            // Read a transformation matrix
+            var transformationMatrixFile = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)
+                               + "/Zivid/RobotTransform.yaml";
+            Console.WriteLine("Getting PoseState:");
+            var transformationMatrix = readTransform(transformationMatrixFile);
```

Comment:
I suggest you make this part equivalent to the same part in the [C++ sample](https://github.com/zivid/zivid-cpp-samples/blob/master/source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp). This means that we want a function called "getTransformationMatrixFromYAML". Practically, put lines 24-26 inside "readTransform" and rename "readTransform".

Reply:
Then I suggest we use comment instead of printing "Getting PoseState:", to be consistent of printing outside functions and comment inside functions?
Think we should do the same in Cpp sample.